spec.html

```diff
@@ -10724,18 +10724,18 @@
     <emu-grammar>
       IdentifierReference[Yield, Await] :
         Identifier
-        [~Yield] `yield`
-        [~Await] `await`
+        `yield`
```

Comment:
Oh, thanks for pointing this out. It seems like the reduce-reduce conflict only really comes up for IdentifierReference, not the other two. I only threw in IdentifierReference here for symmetry, and it's not needed for the actual change here. Would backing out that part be enough?